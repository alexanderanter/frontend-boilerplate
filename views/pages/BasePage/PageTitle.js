import styled from 'styled-components';

const PageTitle = styled.h1`
  text-align: center;
  text-transform: capitalize;
`;

export default PageTitle;
