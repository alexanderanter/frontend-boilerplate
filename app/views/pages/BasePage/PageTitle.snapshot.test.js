import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import PageTitle from './PageTitle';

describe('PageTitle', () => {
  let props;
  let wrapper;

  const Wrapper = () => {
    if (!wrapper) {
      wrapper = shallow(<PageTitle {...props}>{props.children}</PageTitle>);
    }
    return wrapper;
  };

  // Reset Component and Props before each test.
  beforeEach(() => {
    props = {};
    wrapper = undefined;
  });

  describe('without props', () => {
    it('Should match snapshot', () => {
      expect(toJson(Wrapper())).toMatchSnapshot();
    });
  });
});
