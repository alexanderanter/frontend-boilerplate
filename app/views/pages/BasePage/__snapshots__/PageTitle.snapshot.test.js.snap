// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle without props Should match snapshot 1`] = `
.c0 {
  text-align: center;
  text-transform: capitalize;
}

<h1
  className="c0"
/>
`;
